neurots/extract_input/input_parameters.py

```diff
@@ -74,7 +74,7 @@ def merged_params(data):
                 "growth_method": method,
                 "branching_method": branching,
                 "modify": None,
-                "step_size": {"norm": {"mean": 1.0, "std": 0.2}},
```

Comment:
if we have noise here, there will be some imbalance. I am not sure why we would need random step_size, I guess uniform one with 1 micron is also ok, no? If not, we can either leave original value here, but we need to fix to 0 for hipp cells. I would be more in favor of changing default, in case other users bumped into the same issue.